jobs/src/main/scala/dcos/metronome/utils/state/IO.scala

```diff
@@ -0,0 +1,131 @@
+package dcos.metronome.utils.state
+
+import java.io._
+import java.math.BigInteger
+import java.nio.file.{ Files, Path, Paths }
+import java.security.{ DigestInputStream, MessageDigest }
+import java.util.zip.{ GZIPInputStream, GZIPOutputStream }
+
+import com.google.common.io.ByteStreams
+
+import scala.annotation.tailrec
+import scala.util.{ Failure, Success, Try }
+
+object IO {
```

Comment:
This is very similar to `mesosphere.marathon.io.IO`. Why not use that?

Reply:
I just pulled everything from 1.3 in here. I would be inclined not to touch it :)